### PR TITLE
A fix for the library.

### PR DESCRIPTION
**Pull request by ghost**
The library asks for an exchange code when the captcha function isn't there and the reputation endpoint returns arkose or ReCaptcha.

There's 4 other commits, but they were reverted.

### CODE REVIEW

File: src/Account/Auth.js [hunk 1]
```diff
@@ -138,6 +138,17 @@ class AccountAuth {
       });
       await this.authenticate();
       await this.analytics();
+      if(data.verdict === 'arkose' || 'recaptcha' && !credentials.captcha) {
```
Comment (ghost) [inline src/Account/Auth.js]: the data object is not defined.

Comment (ghost) [reply src/Account/Auth.js]: Alright will check out it.

Comment (ghost) [reply src/Account/Auth.js]: I have now fixed it.

**Review by zImPatrick — CHANGES_REQUESTED** [review]
Would it be possible, that you could pass in the exchange token when creating the Launcher object?
Like:
```
const launcher = new Launcher({ 
  email: "some@email.com", 
  password: "password",
  exchangeToken: "93fskot4360"
});
```
edit: fixed formatting

**Comment by SzymonLisowiec** [comment]
My comment to this PR is here: https://github.com/SzymonLisowiec/node-epicgames-client/issues/83#issuecomment-608414704
In addition this PR would broke methods like `enableTwoFactor` while someone would use `exchangeToken`.

Possibility to use `exchangeCode` to login has been added in [`2.0.23`](https://github.com/SzymonLisowiec/node-epicgames-client/commit/ef2c219cf2a7f6df88c1cfc4ec3a488f315ecaf2)

**Pull request closed without merging**